modules/get-started/pages/quickstarts/helm-chart.adoc

```diff
@@ -0,0 +1,369 @@
++
+[source,bash]
+----
+kubectl create configmap connect-input --from-literal=input-data='{"name": "Test User", "email": "test.user@example.com"}' --namespace <namespace>
+----
++
+This ConfigMap will act as the source for the file-based input in Redpanda Connect, allowing the pipeline to read and process this structured JSON data.
+
+. Create a `pipeline.yaml` file.
++
+This file will be used to override the default Helm chart values for Redpanda Connect.
+
+. Add the following configuration to `pipeline.yaml` to set up a simple standalone pipeline that reads data from the file mounted by the ConfigMap, processes it by converting usernames to uppercase, and outputs the result to stdout:
++
+.`pipeline.yaml`
+[source,yaml]
+----
+extraVolumes:
+  - name: input-config
+    configMap:
+      name: connect-input
+extraVolumeMounts:
+  - name: input-config
+    mountPath: /input
+    subPath: input-data
+config:
+  input:
+    file:
+      paths:
+        - "/input" <1>
+  pipeline:
+    processors:
+      - mapping: |
+          root.name = this.name.uppercase() <2>
+  output:
+    stdout: {} <3>
+----
++
+<1> For the input, use the contents of the file at the path where the ConfigMap data is mounted.
+<2> Convert the name field in the input JSON to uppercase using xref:guides:bloblang/walkthrough.adoc[Bloblang], the native mapping language of Redpanda Connect.
+<3> Output the processed data to stdout, making it viewable in logs.
+
+. Deploy the pipeline:
++
+[source,bash]
+----
+helm upgrade --install redpanda-connect redpanda/connect --namespace <namespace> --values pipeline.yaml
+----
+
+. Check the logs:
++
+[,bash]
+----
+export POD_NAME=$(kubectl get pods --namespace <namespace> -l "app.kubernetes.io/name=redpanda-connect,app.kubernetes.io/instance=redpanda-connect" -o jsonpath="{.items[0].metadata.name}")
+kubectl logs $POD_NAME --namespace <namespace>
+----
++
+You should see the username converted to uppercase in the output:
++
+[,json]
+----
+{"name":"TEST USER"}
+----
+
+. Check the Pod's status:
++
+[source,bash]
+----
+kubectl get pods --namespace <namespace> -l app.kubernetes.io/name=redpanda-connect --watch
+----
++
+The Pod enters a `CrashLoopBackOff` state because containers are expected to run continuously. When Redpanda Connect finishes processing the input file and has no further data to process, the Pod exits, causing Kubernetes to restart it repeatedly.
++
+To prevent this status, configure Redpanda Connect to continue processing data indefinitely. One way to do this is by using a `generate` input instead of a static input file. The `generate` input allows Redpanda Connect to continuously produce synthetic data, simulating a live data source.
+
+. Update the `pipeline.yaml` file to use the `generate` input to produce data every second.
++
+[source,yaml]
+----
+config:
```

Comment:
I assume this is the example you are referencing that generates input continuously? If so, is there also a way to suggest how folks can create an even more minimal example for their own use?